src/conversions/either.rs

```diff
@@ -26,28 +26,31 @@
 //!
 //! ```rust
 //! use either::Either;
-//! use pyo3::{Python, ToPyObject};
+//! use pyo3::{Python, PyResult, conversion::IntoPyObject, types::PyAnyMethods};
```

Comment:
Should we add `IntoPyObject` at the top-level, and tidy up examples of imports?

Reply:
I also thought about it multiple times already and given that `ToPyObject` and `IntoPy` are also top-level I think it makes sense to give `IntoPyObject` the same status. Will adapt 👍